src/lib/Vulcan.js

```diff
@@ -11,9 +11,7 @@
  *
  **************************************************************************************************/
 
-/** CEP-Vulcan.js v11.1.0 - This JS Bindings for Vulcan is compatible with Vulcan v7.0 */
-
-/**
+/** Vulcan - v11.2.0 */
```

Comment:
and here we were at 11.1, not at 11.2, wth?

Reply:
https://github.com/Adobe-CEP/CEP-Resources/tree/master/CEP_11.x
these are the they files from CEP_11. I've ran formater (because I've originaly ran that on these files also), and this is the only change for that file

Reply:
same for, CSInterface. not many changes. Only formating changes